src/go/rpk/pkg/cli/cmd/common/common.go

```diff
@@ -478,16 +520,22 @@ func AddKafkaFlags(
 		),
 	)
 
-	AddTLSFlags(command, certFile, keyFile, truststoreFile)
+	AddTLSFlags(command, enableTLS, certFile, keyFile, truststoreFile)
 
 	return command
 }
 
 func AddTLSFlags(
-	command *cobra.Command, certFile,
-	keyFile,
-	truststoreFile *string,
+	command *cobra.Command,
+	enableTLS *bool,
+	certFile, keyFile, truststoreFile *string,
 ) *cobra.Command {
+	command.PersistentFlags().BoolVar(
+		enableTLS,
+		enableTLSFlag,
+		false,
+		"Enable TLS for the Kafka API.",
```

Comment:
`Enable TLS for the Kafka API with system certs (not necessary if specifying custom certs)`?

Reply:
Good suggestion. Changed!